sdks/python/apache_beam/examples/complete/check_requirements_file_packages.py

```diff
@@ -0,0 +1,71 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
this is a test, so let's not have to keep it in examples. 
runners/portability/stager_it_test.py may be a better home for it.

Reply:
Yes, I will move this test and the test which uses setup.py in a different jira ticket.